ci-operator/step-registry/layered-product/odf/aws/add-storage/layered-product-odf-aws-add-storage-commands.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -x
+set -o nounset
+set -o errexit
+set -o pipefail
+
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+
+# create the policies namespace
+oc apply -f - <<EOF
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-storage
+EOF
+
+oc label namespace openshift-storage openshift.io/cluster-monitoring=true
+
+# create 3 machinesets for ocp storage on aws
+CLUSTERID=$(oc get machineset -n openshift-machine-api -o jsonpath='{.items[0].metadata.labels.machine\.openshift\.io/cluster-api-cluster}')
+echo $CLUSTERID
+
+curl -s https://raw.githubusercontent.com/red-hat-storage/ocs-training/master/training/modules/ocs4/attachments/cluster-workerocs-us-east-2.yaml | sed -e "s/CLUSTERID/${CLUSTERID}/g" | oc apply -f -
+
+
```

Comment:
maybe need to sleep few mins cuz it takes some times to make nodes up

Reply:
The code on lines 27-36 is meant to be a loop to wait for the nodes to be up, though its not perfect it'll complete once the first node appears to be running.